### PR TITLE
remove bom and put to tmp folder

### PR DESCRIPTION
**Pull request by shmuelko**

### CODE REVIEW

File: src/practitest_firecracker/core.clj [hunk 1]
```diff
@@ -29,17 +29,16 @@
     (if exit-message
       (exit (if ok? 0 1) exit-message)
       (do
-        (remove-bom (file (first (:reports-path options))))
+        (doall (map remove-bom (:reports-path options)))
```
Comment (stask) [inline src/practitest_firecracker/core.clj]: If you need to do something for each element of a collection, you can use `doseq`.
```clojure
(doseq [path (:report-paths options)]
  (remove-bom path))
```

File: src/practitest_firecracker/core.clj [hunk 2]
```diff
@@ -29,17 +29,16 @@
...
         (let [client             (make-client (select-keys options [:email :api-token :api-uri :max-api-rate]))
-              reports            (parse-reports-dir (:reports-path options))
-              directory          (file (first (:reports-path options)))
-              additional-reports (send-directory directory reports)]
+              reports            (parse-reports-dir (map #(str % "/tmp") (:reports-path options)))
```
Comment (stask) [inline src/practitest_firecracker/core.clj]: Regarding `tmp` -- see my comment in the PR 10 of `test-xml-parser`.

**Review by stask — CHANGES_REQUESTED** [review]

File: src/practitest_firecracker/core.clj [hunk 3]
```diff
@@ -24,37 +24,45 @@
...
     (if exit-message
       (exit (if ok? 0 1) exit-message)
       (do
-        (remove-bom (file (first (:reports-path options))))
+        (doseq [report-path (:reports-path options)]
+          (remove-bom report-path))
         (let [client             (make-client (select-keys options [:email :api-token :api-uri :max-api-rate]))
-              reports            (parse-reports-dir (:reports-path options))
-              directory          (file (first (:reports-path options)))
+              directory          (map #(.getAbsolutePath (file "tmp" %)) (:reports-path options))
```
Comment (stask) [inline src/practitest_firecracker/core.clj]: Temp folder should be passed as a parameter. You might not have permissions to reports path and will not be able to create the "tmp" subfolder.

**Review by stask — CHANGES_REQUESTED** [review]

File: src/practitest_firecracker/core.clj [hunk 4]
```diff
@@ -24,37 +24,45 @@
        (println (str ~module " elapsed time: " (/ (double (- (. System (nanoTime)) start#)) 1000000.0) " msecs")))
      ret#))
 
+(defn clean-tmp-folder [directory]
+  (doseq [path directory]
+    (delete-recursively! path)))
```
Comment (stask) [inline src/practitest_firecracker/core.clj:29]: Did you test this part? Are you sure `directory` is an array of paths?

**Review by stask — CHANGES_REQUESTED** [review]

File: src/practitest_firecracker/core.clj (hunk 4 above)
Comment (shmuelko) [reply src/practitest_firecracker/core.clj:29]: it works, I tested it.
directory comes from: 
`directory          (map #(.getAbsolutePath (file (:temp-folder options) %)) (:reports-path options))`

**Review by stask — APPROVED** [review]

**Pull request merged**